app/src/main/java/pl/brightinventions/slf4android/Slf4AndroidLogFileProvider.java

```diff
@@ -0,0 +1,12 @@
+package pl.brightinventions.slf4android;
+
+import android.content.Context;
+import android.support.v4.content.FileProvider;
+
+public class Slf4AndroidLogFileProvider extends FileProvider {
```

Comment:
Does this class have to be public? It's probably not a big deal if it is, still I'm curious...

Reply:
Yes, it does.